redfish-core/lib/pcie.hpp

```diff
@@ -35,10 +35,10 @@
 namespace redfish
 {
 
-static constexpr const char* pcieDeviceInterface =
-    "xyz.openbmc_project.Inventory.Item.PCIeDevice";
-static constexpr const char* pcieSlotInterface =
-    "xyz.openbmc_project.Inventory.Item.PCIeSlot";
+static constexpr std::array<std::string_view, 1> pcieDeviceInterface = {
+    "xyz.openbmc_project.Inventory.Item.PCIeDevice"};
+static constexpr std::array<std::string_view, 1> pcieSlotInterface = {
+    "xyz.openbmc_project.Inventory.Item.PCIeSlot"};
```

Comment:
Since these may be (are) referenced in other places - e.g. fabric_adapters.hpp, pcie_slots.hpp etc, can they be located in `utils/pcie_util.hpp`, or in a new utils (e.g. utils/interface_utils.hpp)?

Or, we can do those refactoring later...